os/fs/vfs/fs_stat.c

```diff
@@ -64,71 +64,14 @@
 #include "inode/inode.h"
 
 /****************************************************************************
- * Private Functions
+ * Preprocessor definitions
  ****************************************************************************/
+#define RESET_BUF(b) do { memset((b), 0, sizeof(*(b))); } while (0)
```

Comment:
I added comment about this, It is not useful because only 1 line using this.
Also code also not good, I think

Reply:
Do you want this removed?

Reply:
yes I do.

Reply:
Done.